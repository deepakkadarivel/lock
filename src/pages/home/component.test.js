import React from 'react';
import { create } from 'react-test-renderer';
import Home from './component';

describe('Home Comoponent', () => {
    const doors = [
        { id: 1, name: 'door one' },
        { id: 2, name: 'door two' },
        { id: 3, name: 'door two' }
    ];
    const user = {
        id: 1,
        name: 'user one',
        accessibleDoors: doors
    };
    const props = {
        loadPeople: () => {},
        loadDoors: () => {},
        setAppUser: () => {},
        checkAccess: () => {},
        people: [user],
        doors,
        user: {}
    };

    test('Render Home compoent', () => {
        const component = create(<Home {...props} />);

        expect(component.toJSON()).toMatchSnapshot();
    });
});
