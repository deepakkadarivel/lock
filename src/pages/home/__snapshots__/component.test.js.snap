// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Comoponent Render Home compoent 1`] = `
<div
  className="Home"
>
  <h1>
    Unlock Doors
  </h1>
  <div
    className="Doors"
  >
    <div
      className="Doors-item"
      onClick={[Function]}
      role="presentation"
    >
      <i
        className="material-icons"
      >
        meeting_room
      </i>
      door one
    </div>
    <div
      className="Doors-item"
      onClick={[Function]}
      role="presentation"
    >
      <i
        className="material-icons"
      >
        meeting_room
      </i>
      door two
    </div>
    <div
      className="Doors-item"
      onClick={[Function]}
      role="presentation"
    >
      <i
        className="material-icons"
      >
        meeting_room
      </i>
      door two
    </div>
  </div>
  <button
    className="Fab ripple"
    onClick={[Function]}
    type="button"
  >
    <i
      className="material-icons"
    >
      person_outline
    </i>
  </button>
</div>
`;
